mmdet/core/bbox/match_costs/match_cost.py

```diff
@@ -246,10 +246,11 @@ class DiceCost:
         eps (float, optional): default 1e-12.
     """
 
-    def __init__(self, weight=1., pred_act=False, eps=1e-3):
+    def __init__(self, weight=1., pred_act=False, eps=1e-3, solo_style=False):
         self.weight = weight
         self.pred_act = pred_act
         self.eps = eps
+        self.solo_style = solo_style
```

Comment:
solo_style is not an appropriate name. You may check the name of option in dice loss. https://github.com/open-mmlab/mmdetection/blob/master/mmdet/models/losses/dice_loss.py